core/src/rpc.rs

```diff
@@ -1766,6 +1821,29 @@ impl RpcSol for RpcSolImpl {
         meta.get_account_info(&pubkey, config)
     }
 
+    fn get_multiple_accounts(
+        &self,
+        meta: Self::Metadata,
+        pubkey_strs: Vec<String>,
+        config: Option<RpcAccountInfoConfig>,
+    ) -> Result<RpcResponse<Vec<Option<UiAccount>>>> {
+        debug!(
+            "get_multiple_accounts rpc request received: {:?}",
+            pubkey_strs.len()
+        );
+        if pubkey_strs.len() > NUM_LARGEST_ACCOUNTS {
```

Comment:
Let's not reuse `NUM_LARGEST_ACCOUNTS`.  I don't think somebody would expect changing that constant to affect this method.  A new `MAX_MULTIPLE_ACCOUNTS` perhaps?

Reply:
No, definitely not. Sorry that wasn't clear. I just threw that in as a starting point, until I get more information about that req.